scripts/train/run_batch.sh

```diff
@@ -98,6 +128,13 @@ while [[ -z "$EXIT_CODE" ]]; do
       break
     fi
   fi
+
+  if [[ "${RESULT}" == "started" && "$monitoring_started" != true  ]]; then
+    echo "Training started, start monitoring."
+    (python3 training_cluster_monitor.py --project_id=${PROJECT} --checkpoint_path=${MODEL_OUTPUT}/${MODEL_CHECKPOINT}_checkpoints) &
```

Comment:
The MODEL check point can't be used as is as it sometimes has '/' or it could be a full GCS path. Check   https://github.com/GoogleCloudPlatform/llm-pipeline-examples/blob/12fd0cb0812cfaead860ba0d0ee4163b1085c42f/src/finetune.py#L95

I suggest we move the code from the line I referenced to a function and reuse it inside training_cluster_monitor.py

Reply:
What's the list of acceptable model checkpoints in addition to a single name or a full GCS path?

Reply:
It's either a GCS path or a  hugging face model check point. hugging face checkpoints are either 'name' or 'company/name'

Reply:
Updated to search for model checkpoint files with regex instead of rely on directory name.